README.md

```diff
@@ -152,7 +152,7 @@ Atomic Layout's browser support is made by the browser support of underlying tec
 
 Thank you for deciding to contribute! Your involvement makes a significant impact on the library and its future.
 
-**Please read the [Contribution guidelines](./.github/CONTRIBUTING.md)** to get familiar with the contributing process. The issues labeled [`help wanted`](https://github.com/kettanaito/atomic-layout/labels/help%20wanted) or [`good first issue`](https://github.com/kettanaito/atomic-layout/labels/good%20first%20issue) are a good place to start cooperating on Atomic Layout. Feature suggestions or bug reports, discussion, and pull requests are always welcome!
+**Please read the [Contribution guidelines](https://github.com/kettanaito/atomic-layout/blob/277-broken-README.md/.github/CONTRIBUTING.md)** to get familiar with the contributing process. The issues labeled [`help wanted`](https://github.com/kettanaito/atomic-layout/labels/help%20wanted) or [`good first issue`](https://github.com/kettanaito/atomic-layout/labels/good%20first%20issue) are a good place to start cooperating on Atomic Layout. Feature suggestions or bug reports, discussion, and pull requests are always welcome!
```

Comment:
It seems you have included the absolute link relative to the feature branch. Please, could you include it relative to master branch (just as you did with other assets above)?

Reply:
@kettanaito  i did that already